zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.zookeeper.server.backup;
+
+public class BackupConstants {
```

Comment:
Please see my reply here: https://github.com/linkedin/zookeeper/pull/20/files#r558721414

In short, is this used throughout the whole backup logic or only used in the scope of NfsBackupStorage?

Reply:
For this case, I think it will be used for the broader scope because no matter which backup storage we are using, we want to unify how we represent a existing file in BackupFileInfo.

Reply:
In that case, would BackupFileInfo be the right place for these constants instead of creating a whole constant class? Or do you still think we should keep them in a separate constant file?

Also, we should rename constants so they make more sense:

NOT_FOUND_FILE_MODIFIED_TIME -> FILE_MODIFIED_TIME_NOT_FOUND
NOT_FOUND_FILE_SIZE -> FILE_SIZE_NOT_FOUND

I also recommend using one constant for some field not being found in a file. It looks like what you really want to do is to create codes for different error scenarios, https://stackoverflow.com/questions/446663/best-way-to-define-error-codes-strings-in-java explains a way to do this. But this might be over-engineering for now, so I'd be okay with a simple FIELD_NOT_FOUND.